credentials/credentials.go

```diff
@@ -126,6 +126,10 @@ func (t TLSInfo) AuthType() string {
 	return "tls"
 }
 
+func (t TLSInfo) ConnState() tls.ConnectionState {
```

Comment:
I missed this in my previous PR. I actually tend to make state exported directly.